configure.ac

```diff
@@ -195,10 +195,12 @@ AC_ARG_ENABLE([upstart],
     [enable_upstart=try],
 )
 
-dnl FIXME currently unused
 AC_ARG_ENABLE([systemd],
     [  --enable-systemd
-       Enable support for the systemd init system [default=try]])
+       Enable support for the systemd init system [default=try]],
```

Comment:
Something like "Enable support for managing resources via systemd" might be more accurate (and similar for upstart)